resources/schema.py

```diff
@@ -151,6 +151,12 @@ class Meta:
 
 
 class PierNode(graphql_geojson.GeoJSONType):
+    number_of_free_places = graphene.Int(required=True)
+    number_of_places = graphene.Int(required=True)
+    max_width = graphene.Float()
+    max_length = graphene.Float()
```

Comment:
I think this two are non-nullable non-blankable, so there will always be values

Reply:
ah, but if there are no places/berths... right

Reply:
Yeah, if there are no places, it will return `null`. 👍